docs2/site/docs/getting-started/user-context.md

```diff
@@ -28,3 +28,20 @@ public class Query : ObjectGraphType
   }
 }
 ```
+
+If you need to access the User from the http request and populate this globally, you will need to add a User property to your custom `UserContext` class and modify your Startup configuration as follows:
```

Comment:
> populate this globally

What does _globally_ mean?

Reply:
Thank you for catching this, adjusted the wording and added the other changes suggested.